src/launcher/java/org/truffleruby/launcher/Launcher.java

```diff
@@ -241,12 +241,10 @@ private static void printTruffleMemoryMetric() {
 
     public static String getVersionString(boolean isGraal) {
         return String.format(
-                "%s %s, like ruby %s (%s revision %s) <%s %s %s> [%s-%s]",
+                "%s %s, like ruby %s <%s %s %s> [%s-%s]",
```

Comment:
I much preferred the one with the revision date. Why back this out?

Reply:
The revision SHA-1 is already in the engine version (shown as the second `%s`), unless it's a GraalVM release and then we have a git tag for it.

Reply:
I missed that the SHA is part of the `ENGINE_VERSION`. So I think my comment can be ignored.